tasks/install.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Ensure hostname is resolvable
```

Comment:
I'm not sure to be honest, we have it in both for the gocd roles, wouldn't having stuff other than just includes in main.yml go against our current conventions?